target/linux/ar71xx/base-files/lib/ar71xx.sh

```diff
@@ -287,11 +287,16 @@ tplink_board_detect() {
 	"360000"*)
 		model="TP-Link TL-WDR3600"
 		;;
-	"3C0001"*)
+	"3c0001"*)
 		model="OOLITE"
+		[ "$hwid" = "3c00010c" -a "$mid" = "00000001" ] && {
+			model="MINIBOX_V3.2"
+			hwver=""
```

Comment:
set it to v3.2

Reply:
This was my fault. I supplied these lines when responding to a request for help programmatically differentiating this device from the Oolite and the MiniBox v1. On second glance, it appears this code may not be even called, because MIPS_MACHINE does not yield a string having "TP-LINK " in front of it. IIRC, it had some use for the OEM firmware's flashing function.